morphic.js

```diff
@@ -9403,16 +9439,34 @@ MenuItemMorph.prototype.mouseEnter = function () {
         this.image = this.highlightImage;
         this.changed();
     }
+    if (this.doubleClickAction instanceof MenuMorph) {
+        var w = this.world();
```

Comment:
As it turns out the variable was used only once, so I flushed it altogether.  Some other time I'll discuss with you why one-letter names other than x are perfectly fine when correctly used. :)